### ISSUE TITLE
wav2vec training time & meaning of the log output.

### ISSUE THREAD

**Issue by joazoa** [pytorch/fairseq#1955]
Hello,

I running wav2vec on ~10.000 hours of audio on 6 GPU's with the aim of trying it with wav2letter++
I read in the papers that the 2 models were trained on 40.000 and 400.000 steps.

When should or can I stop training ? When I reach a certain loss on an epoch ? When I reach 400.000 num_updates ? when i reach 400.000 / number of cards ?

What is the meaning of loss, wps, ups, wpb, etc ?

81989/88743 [10:51:46<53:53,  2.09it/s, loss=5.72711e-14, wps=8.90638e+19, ups=2.11, wpb=4.2243e+19, bsz=5.26836e+06, num_updates=348000, lr=0.00020708, gnorm=0, clip=0, loss_scale=4, train_wall=47, wall=166952]
